src/main/java/in/ac/bits/protocolanalyzer/mvc/controller/TestController.java

```diff
@@ -38,24 +40,22 @@
 	@Qualifier("Experiment")
 	private Experiment exp1;
 
-//	@RequestMapping(method = RequestMethod.GET)
-	public void runSequentialExp()throws Exception {
-		String pcapPath = "/home/vikram/Darshini/Test_pcap_files/sample_capture_test1.pcap";
-		String protocolGraphStr = "graph start {\n\tethernet;\n}\ngraph ethernet {\n\tswitch(ethertype) {\n\t\tcase 0800:			 ipv4;\n\t}\n}\ngraph ipv4 {\n\tswitch(protocol) {\n\t\tcase 06: tcp;\n\t}\n}\ngraph tcp {\n}\ngraph end {\n}";
-
+	@RequestMapping(value = "/experiment", method = RequestMethod.GET)
+	public @ResponseBody String runSequentialExp(@RequestParam("protocolGraphPath") String protocolGraphPath, @RequestParam("pcapPath") String pcapPath)throws Exception {
+	    String protocolGraphStr = new String(Files.readAllBytes(Paths.get(protocolGraphPath)));
+	    
 		exp1.init(pcapPath, protocolGraphStr);
-		exp1.analyze();
+		return exp1.analyze();
+		
 		/*
 		TimeUnit.SECONDS.sleep(10);
 		log.info("] LAUNCHING EXPERIMENT TWO");
 		exp2.init(pcapPath, protocolGraphStr);
 		exp2.call();*/
 	}
 
-//	@RequestMapping(value = "/experiment", method = RequestMethod.GET)
-  @RequestMapping(method = RequestMethod.GET)
+	@RequestMapping(method = RequestMethod.GET)
 	public @ResponseBody String runExp(@RequestParam("pcapPath") String pcapPath)throws Exception {
-		//String pcapPath = "/home/vikram/Darshini/Test_pcap_files/sample_capture_test1.pcap";
 		String protocolGraphStr = "graph start {\n\tethernet;\n}\ngraph ethernet {\n\tswitch(ethertype) {\n\t\tcase 0800:			 ipv4;\n\t}\n}\ngraph ipv4 {\n\tswitch(protocol) {\n\t\tcase 06: tcp;\n\t}\n}\ngraph tcp {\n}\ngraph end {\n}";
 
 		exp1.init(pcapPath, protocolGraphStr);
```

Comment:
The statement `return exp1.analyze();` does not make the value being returned very apparent. It might be better to store the value into an intermediate variable of appropriate name and type. We can then return the value in the intermediate variable.

The commented section of the code following return statement can also be removed.